src/data_structures/hashmap.md

```diff
@@ -0,0 +1,20 @@
+### Hashmaps
+
+With arrays you have to store data in an ordered sequence identified with a indx. `Hashmaps` store data in a (key, pair) sequence that you can access by the index referring to another Object(indx is also an Object)
+
+```java
+import java.util.HashMap;
+HashMap<Type, Type> Name = new HashMap<Type, Type>();
```

Comment:
If its `<Type, Type>` , that implies that both the keys and the values are the same type? Thats confusing.